docs/source/how-to/manage-environments.rst

```diff
@@ -62,6 +62,12 @@ in-place, include:
 - cluster install method (e.g., `parcels`_): depending on your infrastructure, there may be
   ways to install specific binaries to all workers in a cluster.
 
+There are also tools that replicate your local machine's environment in the cloud. If you are
+using `Coiled <https://www.coiled.io/>`_, for example, your local Python environment is automatically
+replicated in your Dask cluster in the cloud (see the `Coiled documentation on package sync <https://docs.coiled.io/user_guide/package_sync.html>`_).
```

Comment:
I like this with my Coiled hat on, I'm not sure how I feel with my community hat.  I'd defer to Jacob on this one.

Reply:
I'm fine with this. It is a statement of fact and is mentioned in a footnote style below what I would consider to be the "Dask Community" bit.

Reply:
I actually decided to remove this section, @gjoseph92 added it to the Deployment Considerations page in #9933 and I think it's a clearer fit there